Samples/Tools/CMakeExample/XboxOne/CMakeLists.txt

```diff
@@ -55,13 +55,16 @@ foreach(FILE ../VertexShader.hlsl ../PixelShader.hlsl)
 endforeach(FILE)
 
 # Microsoft Config File
+set(TDF_NAME "XboxOne")
+configure_file(../MicrosoftGameConfig.mgc MicrosoftGame.config @ONLY)
```

Comment:
I forgot to add the new `MicrosoftGameConfig.mgc` to distribution. I'll fix this shortly.